packages/fxa-auth-server/lib/subhub/client.js

```diff
@@ -183,16 +183,14 @@ module.exports = function (log, config) {
           orig_system: ORIG_SYSTEM
         });
       } catch (err) {
-        if (err.statusCode === 400) {
+        if (err.statusCode === 400 || err.statusCode === 402 || err.statusCode === 404) {
```

Comment:
Do we still need this `if` condition given the child conditions below? Only consequence of removing it would seem to be that we call `log.error` for all errors, which may not be a bad thing anyway.